packages/extension-chakra-storefront/src/components/product-tile/index.jsx

```diff
@@ -179,10 +181,10 @@ const ProductTile = (props) => {
     }, [product, badgeDetails])
 
     return (
-        <Box {...styles.container}>
-            <Link data-testid="product-tile" to={productUrl} {...styles.link} {...rest}>
-                <Box {...styles.imageWrapper}>
-                    <AspectRatio {...styles.image}>
```

Comment:
FYI: I just renamed styles.image to styles.aspectRatio as I had to add addition styling specific to the image. See theme/components/project/product-tile.js